{{cookiecutter.project_slug}}/README.md

```diff
@@ -6,19 +6,31 @@ This repository contains a [`FastAgency`](https://github.com/airtai/fastagency)
 
 To run this [`FastAgency`](https://github.com/airtai/fastagency) application, follow these steps:
 
-1. Open this folder with [vscode](https://code.visualstudio.com/).
+1. To run the `FastAgency` application, you need an [OpenAI API Key](https://openai.com/index/openai-api/). Create one and set it as an environment variable in the terminal using the following command:
```

Comment:
This is not true. You need a key for some kind of LLM API.

Reply:
We should reword it in such a way to say this is the most common way to use FastAgency/AutoGen, but not the only and include this link: https://fastagency.ai/latest/user-guide/runtimes/autogen/using_non_openai_models/